config.yaml.dist

```diff
@@ -4,13 +4,29 @@ probes:
 #    client_cert: /path/to/cert.pem # Use either user/pass auth, certificates or both at the same time.
 #    client_key: /path/to/cert.key
 #    ca_chain: /path/to/ca-chain.pem
+#    insecure_skip_verify: false
```

Comment:
Please add sth like

```
# Skip TLS verification
```

to the end of the line. Just like in line 4 (client_cert). I think insecure_skip_verify is a very generic name..

Reply:
ah, see line 21. just copy the one from below. :)

Reply:
I'll provide an updated ``config.yaml.dist`` file